README.md

```diff
@@ -137,4 +132,6 @@ has configuraiton parameters specific to its platform. These deployers are:
 
 There is also a
 [Python deployer](https://github.com/arcalot/arcaflow-engine-deployer-python) that
-allows for running supported plugins directly instead of containerized.
+allows for running Python plugins directly instead of containerized. *Note that not all
+Python plugins may work with the Python deployer, and any plugin dependencies must be
+on the target system.*
```

Comment:
"must be on" is slightly vague.  "must be _installed_ on" (or "present on"), perhaps?